R/expr-string.R

```diff
@@ -194,7 +201,9 @@ expr_str_to_datetime <- function(
   wrap({
     check_dots_empty0(...)
     self$`_rexpr`$str_to_datetime(
-      self, format, time_unit, time_zone, strict, exact, cache, ambiguous
+      format = format, time_unit = time_unit, time_zone = time_zone,
+      strict = strict, exact = exact, cache = cache,
+      ambiguous = as_polars_expr(ambiguous, as_lit = TRUE)$`_rexpr`
```

Comment:
Please check ambiguous as like this:
https://github.com/eitsupi/neo-r-polars/blob/49ac30a9bf6f6d88b3b0725d08b59b75c0244c80/R/series-to_r_vector.R#L191-L194